nano/node/active_transactions.cpp

```diff
@@ -428,7 +428,7 @@ nano::election_insertion_result nano::active_transactions::insert (std::shared_p
 	auto const existing = roots.get<tag_root> ().find (root);
 	if (existing == roots.get<tag_root> ().end ())
 	{
-		if (!recently_confirmed.exists (root))
+		if (!recently_confirmed.exists (root) && !node.confirmation_height_processor.is_processing_or_confirmed (hash))
```

Comment:
There is no need to refer to `node`. nano::active_transactions has `confirmation_height_processor`.
```
!confirmation_height_processor.is_processing_or_confirmed (hash)
```